File: src/main/java/guru/xgm/language/wts/psi/tree/WtsIElement.java
package guru.xgm.language.wts.psi.tree;

import com.intellij.psi.tree.IElementType;
import guru.xgm.language.wts.lang.WtsLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class WtsIElement extends IElementType {

    public WtsIElement(@NotNull @NonNls String debugName) {
        super(debugName, WtsLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "WtsIElement." + super.toString();
    }
}

File: src/main/kotlin/guru/xgm/language/wts/psi/tree/WtsIElement.kt
package guru.xgm.language.wts.psi.tree

import com.intellij.psi.tree.IElementType
import guru.xgm.language.wts.lang.WtsLanguage
import org.jetbrains.annotations.NonNls

class WtsIElement(debugName: @NonNls String) : IElementType(debugName, WtsLanguage.instance) {
    override fun toString(): String {
        return "WtsIElement." + super.toString()
    }
}
